content/spin/variables.md

```diff
@@ -0,0 +1,205 @@
+title = "Configuration Variables"
+template = "spin_main"
+date = "2022-06-14T00:22:56Z"
+enable_shortcodes = true
+[extra]
+url = "https://github.com/fermyon/developer/blob/main//content/spin/variables.md"
+
+---
+- [Using the Config SDK to Get Variables](#using-the-config-sdk-to-get-variables)
+
+Spin supports dynamic application variables. Instead of being static, their values can be updated without modifying the application, creating a simpler experience for rotating secrets, updating API endpoints, and more. 
+
+These variables are defined in a Spin application manifest (in the `[variables]` section) and are provided by a [configuration provider](/spin/dynamic-configuration.md#custom-config-providers). When running Spin locally, the configuration provider can be Vault for secrets or host environment variables. Refer to the [dynamic configuration documentation](/spin/dynamic-configuration.md) to learn how to configure variables locally.
+
+## Using the Config SDK to Get Variables
```

Comment:
The rubric in other API guides has been "How to Use X From Applications" - suggest avoiding "config SDK" as if it was as separate SDK.